extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/TransactionManagerBuildTimeConfig.java

```diff
@@ -0,0 +1,34 @@
+package io.quarkus.narayana.jta.runtime;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.BUILD_TIME)
+public final class TransactionManagerBuildTimeConfig {
+    /**
+     * Allow using multiple XA unaware resources in the same transactional demarcation with XA capable resources.
```

Comment:
```suggestion
     * Allow using multiple XA unaware resources in the same transactional demarcation.
```

It doesn't matter whether other XA resources are involved.